src/index.js

```diff
@@ -1,6 +1,9 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
+
+// testing the home component
+import Home from "./pages/Home"
```

Comment:
I don't think this is needed, only the "import App from './App'

Reply:
removed from original file